docs/src/prototypes/chatMessages/ChatMessageWithPopover/Popover.tsx

```diff
@@ -31,10 +34,20 @@ class Popover extends React.Component<PopoverProps, PopoverState> {
     this.setState({ focused: e.currentTarget.contains(e.relatedTarget) })
   }
 
+  handleActionableItemClick = e => {
+    const { onShowActionMenuChange, chatMessageRef } = this.props
+    onShowActionMenuChange(false)
+    // Currently when the action menu is closed because of some actionable item is clicked, we focus the ChatMessage
+    // this was not in the spec, so it may be changed if the requirement is different
+    e.type === 'keydown' && chatMessageRef && chatMessageRef.focus()
+  }
+
   render() {
+    const { onShowActionMenuChange, onForceShowActionMenuChange, ...rest } = this.props
+    delete rest.chatMessageRef
```

Comment:
nit: maybe just mention this prop while destructuring the object? I think it will be cleaner as all the intents will be reflected by single expression. 

```js
// it is clear from this line what 'rest' is, 
// and we are not mutating this 'rest' object anywhere in the code afterwards
const { onShowActionMenuChange, onForceShowActionMenuChange, chatMessageRef, ...rest } = this.props
```

However, please, feel free to decide what is better :)

Reply:
Why I did this is, the chatMessageRef is not used anywhere in the code, so I am getting an error...